src/NServiceBus.AcceptanceTesting/Support/EndpointRunner.cs

```diff
@@ -135,7 +134,7 @@ public Result Start()
             }
         }
 
-        public Result Stop()
+        public Task<Result> Stop()
```

Comment:
My opinion yes. When the bus later can be shut down asynchronously we need it. You could call YAGNI now but for symmetrical reasons I think when Start returns a task then stop should also return as task.